README.md

```diff
@@ -1031,9 +1034,9 @@ Other Style Guides
     ```
 
   <a name="arrows--one-arg-parens"></a><a name="8.4"></a>
-  - [8.4](#arrows--one-arg-parens) Always include parentheses around arguments for clarity and consistency. eslint: [`arrow-parens`](https://eslint.org/docs/rules/arrow-parens)
+  - [8.4](#arrows--one-arg-parens) Always include parentheses around parameters for clarity and consistency. eslint: [`arrow-parens`](https://eslint.org/docs/rules/arrow-parens)
 
-    > Why? Minimizes diff churn when adding or removing arguments.
+    > Why? Minimizes diff churn when adding or removing parameters.
```

Comment:
while this is technically correct, colloquially the two terms are synonymous, and the eslint docs describe the rule as "Require parentheses around arrow function arguments", so i don't think this change helps much.

Reply:
Yes, but in the next sentences on the [same page](https://eslint.org/docs/latest/rules/arrow-parens) they write

_Arrow functions can omit parentheses when they have exactly one parameter. In all other cases the parameter(s) must be wrapped in parentheses_

From my experience understanding of the difference between arguments and parameters is very important especially for beginners and it's not a good idea to use those terms interchangeably.

So, let's keep arguments?

Reply:
I don't think it's super important - but it's fine to keep this change.